docs/decoupling.md

```diff
@@ -3,16 +3,18 @@ description: Instructions on running WildBeast from source instead of from Docke
 path: tree/master/docs
 source: decoupling.md
 
-This document outlines the procedure for decoupling WildBeast from Docker and running it from source instead, enabling custom commands and other modifications to the behaviour of it.
+This document outlines the procedure for decoupling WildBeast from Docker, allowing the entire system to be leveraged as opposed to a Docker sandbox.
 
 ## Preface
 
-If you have already set up your WildBeast instance in accordance with the available installation guides, feel free to jump to the next section. If you have not done this yet, however, please follow through with that procedure before embarking onto this guide. Most of the steps assume knowledge of what has been done previously, and as such it is highly recommended to follow proper procedure first.
+If you have already set up your WildBeast instance in accordance with the available installation guides, feel free to jump to the next section. If you have not done this yet, however, please follow through with that procedure before embarking onto this guide.
 
-Additionally, be sure to shutdown all existing Docker containers pertaining to WildBeast before proceeding.
+Additionally, consider whether you need to go through the decoupling process to begin with. The Docker container for WildBeast runs the code that is in the WildBeast install directory, and when the container is restarted, the changes are mirrored. However, if you want to use something akin to live reloading (Such as Nodemon) or have other valid reasons, feel free to proceed.
```

Comment:
`WildBeast install directory`  -> WildBeast directory ?

Reply:
Good point, this can be seen as confusing. Correcting.